src/containers/BooksList.js

```diff
@@ -5,12 +5,13 @@ import { removebook } from '../actions/index';
 import BookConatiner from '../components/Book';
 
 const BooksList = ({ books, removebook }) => (
```

Comment:
- [ ] [REQUIRED] it should implement `handleBookRemove function` [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)
- [ ] [REQUIRED] it should pass `handleRemoveBook` to Book component [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)